examples/user-management/sveltekit-user-management/src/app.html

```diff
@@ -3,7 +3,7 @@
 	<head>
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
-		<meta name="viewport" content="width=device-width" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
this one added by mistake ?

Reply:
initial-scale=1.0 part sets the initial zoom level when the page is first loaded by the browser.

Reply:
I meant is it a mistake because it's in our examples directory 😅

Reply:
haha! yes, it is a mistake,wrong directory 😅